apps/web-shop-front/src/pages/CheckoutPage.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { FC, useState } from 'react';
+import { Button, Input, Loader, Image } from '@team21/ui-components'
+import useGetCartProducts from '@team21/web-shop-front/src/graphql/cart/useGetCartProducts';
+import { CartItem } from "@team21/types" 
+import { useUser } from '@team21/web-shop-front/src/state/state';
+import { useForm } from 'react-hook-form';
+import useCreateOrder from '@team21/web-shop-front/src/graphql/order/useCreateOrder'
+import useCreateOrderProducts from '@team21/web-shop-front/src/graphql/order/useCreateOrderProducts'
+import useRemoveItemFromCart from '@team21/web-shop-front/src/graphql/cart/useRemoveItemFromCart'
+
+const CheckoutPage: FC = () => {
+    const [{ user }] = useUser();
+    const { data, loading } = useGetCartProducts(user.cart_id);
+    const { register, handleSubmit, formState } = useForm();
+    const { errors } = formState;
```

Comment:
I think this can be written as:
const { register, handleSubmit, formState: { errors } } = useForm();